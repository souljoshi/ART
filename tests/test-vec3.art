// ** this test should fail but passes **

int main()
{
    vec v1;

    v1 = <10.5, 5.5>;

    v1[0] = 6.7;
    v1[2] = 77.7; // should fail since vec only 2 elements, but doesn't

    printf(v1[0]);
    printf(v1[1]);
    printf(v1[2]); // should fail but doesn't
    printf(v1[5]); // should fail but doesn't
}