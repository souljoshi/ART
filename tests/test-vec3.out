6.700000
5.500000
77.700000
0.000000